cmd/vic-machine/create/create.go

```diff
@@ -499,14 +499,19 @@ func (c *Create) processParams() error {
 		return err
 	}
 
-	if c.BridgeNetworkName == "" {
-		c.BridgeNetworkName = c.DisplayName
+	// prevent usage of special characters for certain user provided values
+	if err := common.CheckUnsupportedChars(c.DisplayName); err != nil {
+		return cli.NewExitError(fmt.Sprintf("--name contains unsupported characters: %s", err), 1)
```

Comment:
I think referring by a parameter name to a problem is good, but feels clunky. Try to rephrase it like "<something> (--name value) contains not allowed characters. Allowed characters are: a-zA-Z, :, ..." Include full set of them, as a user I would love it.